docs-new/docs/guides/evaluatemetrics.md

```diff
@@ -177,7 +177,7 @@ $ kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta2/namespaces/podtato-kube
         "kind": "KeptnMetric",
         "namespace": "keptn-system",
         "name": "keptnmetric-sample",
-        "apiVersion": "metrics.keptn.sh/v1beta1"
+        "apiVersion": "metrics.keptn.sh/v1alpha3"
```

Comment:
The latest version of this library is `v1beta1` so the existing doc is correct.  Please restore that content.